setup.py

```diff
@@ -60,6 +60,7 @@
     'inspire-crawler~=1.0',
     'inspire-dojson~=53.0,>=53.0.0',
     'inspire-matcher~=0.0,>=0.3.3',
+    'inspire-json-merger~=2.0,>=2.0.4',
```

Comment:
Nit: let's keep these requirements alphabetically sorted (this should be above `inspire-dojson`).

Reply:
you mean...below? `d` -> `j`